### ISSUE TITLE
Update a product

### ISSUE THREAD

**Issue by phbe0522** [stern-devops-2020-products/products#4]
**As an** inventory/business manager  

**I need** to update a product  

**So that** I can update the product information and have the correct info reflect on our website  

**Assumptions:** 
* Request include:
= Product name 
= Product description
= SKU
= Price
= ~Discount~ (UPDATE 3/28: We will not include this per discussion between PC/SY)
= Quantity of stock
= Size 
= Color 
= Product category 
= Date and time updated  
* When updating product, validate the product has not already been updated with overlapped information.  For example, indicate to API user that the product had been updated previously within the same day. 

**Acceptance Criteria:**
```
Given a request to update a product
When the products table and column of date updated are queried and the product has not already been updated with the same information
Then the product will be updated (existing columns updated or new columns added)
```

**Comment by sjy252** [comment]
@phbe0522 Hi Phoebe - I'm just finishing this story up and wanted to point something out and clarify another. I noticed you had a "date and time updated" which totally makes sense and I think we should have in here, but I didn't include it as one of the fields we want to capture as part of my first story, but I added it in now. Just making a note of that here. I'm going to post something in Slack quickly asking about the "discount" field and whether it's necessary.
